crates/sound/src/device.rs

```diff
@@ -171,7 +456,7 @@ impl VhostUserBackend<VringRwLock, ()> for VhostUserSoundBackend {
     }
 
     fn max_queue_size(&self) -> usize {
-        256
+        64
```

Comment:
Why this change?

Reply:
Oh I wanted input on this actually. Doesn't this specify the max number of descriptors? (`The number of descriptors in the table is defined by the queue size for this virtqueue: this is the maximum possible descriptor chain length.`) I don't know why, but `crosvm` set this value.

Reply:
Right, I don't know if we have to make it parametric, but maybe the driver in Linux always uses 64 and they put it that way. I might have picked it up from vhost-user-vsock. We need to understand it better, however I don't think it's a big problem.